src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/KestrelThread.cs

```diff
@@ -77,12 +87,6 @@ public Task StartAsync()
             return tcs.Task;
         }
 
-        // This must be called from the libuv event loop.
-        public void AllowStop()
```

Comment:
If you want to be nice, you can go back to keeping this, since #997 will call it again.